artemis-cli/src/main/resources/org/apache/activemq/artemis/cli/commands/bin/artemis-service

```diff
@@ -61,7 +61,7 @@ status() {
     pid=`cat "${PID_FILE}"`
     # check to see if it's gone...
     ps -p ${pid} > /dev/null
-    if [ $? -eq 0 ] ; then
+    if [ $? -eq 0 ] && [ "`pwdx ${pid} | awk '{print $2}'`" == "$ARTEMIS_INSTANCE" ]; then
```

Comment:
Unfortunately it's still broken. I still get the same error as before. You're using `==` when I believe you should be using `=`.